### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by manstie** [chainlist/svelte-forms#88]
**Describe the bug**
The min validator can not be used for a text field where you want n characters and they input a number > n.

**To Reproduce**
1. Add min(8) validator to a text input
2. Type in 9
3. Field is valid

**Expected behavior**
I believe a specific minLength / maxLength validator is required for this situation, as I want the validator to only validate the number of characters in the text and not the potential number value.

**Comment by christoph-jeanluc-schneider** [comment]
I think you ment `2. Type in 7`. But i beleave this bug has been fixed, it works for me (2.3.1).

**Comment by manstie** [comment]
> I think you ment `2. Type in 7`. But i beleave this bug has been fixed, it works for me (2.3.1). 

No, I want to validate the length of the string in a text field, not the numerical value.
